yt_dlp/extractor/instagram.py

```diff
@@ -358,12 +356,29 @@ def _extract_embed_url(webpage):
 
     def _real_extract(self, url):
         video_id, url = self._match_valid_url(url).group('id', 'url')
-        webpage, urlh = self._download_webpage_handle(url, video_id)
+        info = self._download_json(
+            f'https://i.instagram.com/api/v1/media/{_id_to_pk(video_id)}/info/', video_id, fatal=False,
```

Comment:
Are we dropping 2.7 support?

Reply:
2.7? That got dropped ages ago. The minimum supported version of Python is 3.6 (soon it will be 3.7)